comps/embeddings/predictionguard/README.md

```diff
@@ -0,0 +1,20 @@
+# build predictionguard embeddings endpoint docker image
+
+```
+docker build --build-arg http_proxy=$http_proxy --build-arg https_proxy=$https_proxy -t opea/embedding-pg:latest -f comps/embeddings/predictionguard/docker/Dockerfile .
+```
+
+# launch predictionguard embedding endpoint docker container
+
+```
+docker run -d --name="embedding-pg-server" -e http_proxy=$http_proxy -e https_proxy=$https_proxy -p 6000:6000 --ipc=host -e PREDICTIONGUARD_API_KEY=3xbU4Q04lkydH7FIFQ9vWQEg0goIvi opea/embedding-pg:latest
```

Comment:
Looks like this includes an actual key. We will need to:
- Change this to a mask it
- Rotate/ delete this key as it will always be in the commit history now

Reply:
Sorry for that. Key being rotated to new.